doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -538,10 +539,19 @@ Libraries for networking
 
   * Added an error code :c:enumerator:`FOTA_DOWNLOAD_ERROR_CAUSE_INTERNAL` to indicate that the source of error is not network related.
 
+* :ref:`lib_nrf_cloud_rest` library:
+
+  * Replaced the ``nrf_cloud_rest_cell_pos_get`` function with :c:func:`nrf_cloud_rest_location_get`.
+
 * Added:
 
   * :ref:`lib_lwm2m_location_assistance` library that has support for using A-GPS, P-GPS and ground fix assistance from nRF Cloud using an LwM2M server.
 
+* Changed:
+
+  * The nrf_cloud_cell_pos library is now named :ref:`lib_nrf_cloud_location`.
```

Comment:
```suggestion
  * The nRF Cloud cellular positioning library is now renamed to :ref:`lib_nrf_cloud_location`.
```

Reply:
@peknis done, thanks.
it isn't letting me re-request a review from you for some reason.